MuxExoPlayer/src/main/java/com/mux/stats/sdk/muxstats/MuxBaseExoPlayer.java

```diff
@@ -432,7 +449,7 @@ protected void buffering() {
   }
 
   protected void pause() {
-    if (state == PlayerState.SEEKED) {
+    if (state == PlayerState.SEEKED && numberOfPauseEventsSent > 0) {
```

Comment:
This feels a bit risky. Is this necessary? What does it help address?

Reply:
Same as with play event, in general we ignore this event if seeking is in progress, there is a specific case when seek is set but play when ready is false we should send a pause event right after the seeked event, in every other case we should ignore pause event.